src/create-jobs.js

```diff
@@ -146,6 +146,20 @@ function checks (config) {
                 return createQaJobs('./vendor/bin/psalm --shepherd --stats --output-format=github --no-cache', config);
             }
         ),
+        new Check(
+            config.code_checks,
+            [fileTest('composer-require-checker.json')],
+            /**
+             * @param {Config} config
+             * @return {Array}
+             */
+            function (config) {
+                /**
+                 * @see https://github.com/maglnet/ComposerRequireChecker#configuration for config example
```

Comment:
Imho not really useful here as no consumer will ever dive into this project in detail.
Could you link the github repository within the `README` instead? That would be more visible IMO.

Reply:
Okay, I totally remove this as we do not provide any other explanation for other tools.
I'd assume that no one will add infection just because this CI matrix generation supports it.